qa/packages/packages.yaml

```diff
@@ -59,6 +60,7 @@ ceph:
   - rbd-fuse
   - rbd-mirror
   - rbd-nbd
+  - cephfs-mirror
```

Comment:
Why do we need to install this package for every test?

Reply:
I guess I just followed what other daemons were upto. https://github.com/ceph/ceph/pull/36698/files#diff-2093f714b7275b8d49f78ff31202ba21 installs the relevant packages wherever needed.